src/layouts/Layout.astro

```diff
@@ -102,6 +143,63 @@ const {
         text-align: center;
         padding: calc(var(--lh) * 2) var(--lh);
       }
+
+      html.light_mode {
+        .header {
+          box-shadow: inset 0 -1px 0 0 var(--lm-color-border-light);
+          background-color: var(--lm-color-bg);
+        }
+
+        .header__logo path {
+          fill: var(--color-bg);
+        }
+
+        .footer {
+          color: var(--color-bg-contrast);
+        }
+      }
     </style>
+
+    
   </body>
+
+  <script>
```

Comment:
Can we use local storage instead please? We can get rid of these getters/setters then.